browser/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this file boilerplate that we can remove?

Reply:
Good catch this one we can remove